types/mongodb/index.d.ts

```diff
@@ -2,10 +2,11 @@
 // Project: https://github.com/mongodb/node-mongodb-native/tree/2.2
 // Definitions by: Federico Caselli <https://github.com/CaselIT>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
 
 // Documentation : http://mongodb.github.io/node-mongodb-native/2.2/api/
 
+// TypeScript Version: 2.3
```

Comment:
Old location was correct.
Packages that depend on mongo will also have to be updated to have this in their header. #16198 is doing this.